_episodes/07-templates.md

```diff
@@ -1,65 +1,257 @@
 ---
-title: "Page Templates"
+title: "Page Layouts"
 teaching: 0
 exercises: 0
 questions:
 - "How can I design the layout for all pages on my site?"
 - "Where can I find pre-built themes for my site?"
 - "How can I create new layouts based on those I already have?"
 objectives:
-- "Apply a template to a page"
+- "Apply a template layout to a page"
 - "Find and implement pre-existing themes to determine the style of a site"
 - "Create new page templates that inherit from existing layouts"
 keypoints:
 - "Files in the `_layouts/` directory can be used as page templates"
-- "The structure of a page is determined by the `layout` page metadata"
+- "The structure of a page is determined by the `layout` field in the page front matter"
 - "You can find many pre-existing themes for sites on the Internet"
 - "You can avoid duplicated effort by basing new layouts on previous ones"
 ---
 
+Elements that will appear on every page across our website,
+like the banner image we `include`d in our pages in the previous section,
+form part of the structure of the page:
+unlike the content specific to each page,
+these structural elements do not change from page to page.
+As such, although the `include` tags reduce the pain we endure when
+adjusting that repeated content,
+it is not good practice to `include` the same content over and over again.
+This structure can instead be defined within the _layout_ of the pages of the site.
+
+Common elements found in a layout are
+logos,
+banners,
+navigation menus,
+footer content such as copyright and licensing information:
+material often found at the very top or very bottom of a webpage.
+
+## Defining a Layout
+
+To demonstrate how layouts work in Jekyll we will copy the contents of
+`_includes/banner.html`, the HTML file defining the `img` element for our banner,
+to `_layouts/page.html`, a new file in a new folder.
+GitHub doesn't provide a way to make a copy of a file,
+so you will need to do this manually.
```

Comment:
Could we `include` the banner in the page layout instead of copying the code into the page layout?  If not best practice, let me know.

Reply:
Addressed in 64e95810db60b6007029a78034fd3db54e9e99e2